### DISCUSSION TITLE
Cheerio 1.0.0 release date

### DISCUSSION THREAD

**Discussion by DineshSaravan** [cheeriojs/cheerio#2425] [Q&A] [answered]
Hi Team,

Just wondering that when can we expect the next stable release version of `1.0.0`? Which would solve some dependency vulnerability issues.

Thanks.
D

**Comment by fb55** [comment, marked as answer]
The RCs are safe to transition to. I hope to get the final 1.0 release out in the next months, but there is a bit of yak shaving (= work on dependencies) that remains.

**Reply by Bearfinn** [reply]
Hello! It's a year later, so I would like to know how it's going :)
I really love the new API and wish it could be the standard
